README.md

```diff
@@ -46,9 +59,7 @@ Consuming this SDK via Maven is the preferred method of consuming it and using i
 Replace `1.27.4` in `<version>1.27.4</version>` with the latest release version for the SDK.
 Look up the latest SDK version here: https://github.com/aws/aws-iot-device-sdk-java-v2/releases
 
-### Build IoT Device SDK from source
-
-[Install Maven and Set PATH](https://maven.apache.org/install.html)
+### Building AWS IoT SDK from source
```

Comment:
This goes for everything except C++ but maybe we should move the build from source instructions into the FAQ or someone off the main landing page. We could link to it under the "Using SDK" section or something.

Reason being, the only reason why someone would build Java SDK from source is to get something from the main branch that isn't available in a released version yet or if someone is making changes to the SDK directly that they want to use instead of using the SDK as a dependency. This feels like it'd be an incredibly low percentage of people who use our SDK and so we don't want a block on the main landing README that only servers a small handful of people.